common/lib/protocol-definitions/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/protocol-definitions",
-	"version": "2.0.0",
+	"version": "1.2.0",
```

Comment:
@tylerbutler I think this is correct to revert, just want to double-check it will be the right version after all the merging has finished

Reply:
i'm going to merge this to keep the test fixes moving forward, but we can modify this if there is a problem in a follow up.